kolibri/auth/test/test_api.py

```diff
@@ -175,10 +175,6 @@ def test_facility_user_can_get_detail(self):
             'name': self.facility1.name,
         }, dict(response.data))
 
-    def test_anonymous_user_gets_empty_list(self):
```

Comment:
This is no longer true. We return facilities that allow sign ups to anonymous users.